unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1240,5 +1261,37 @@ public static void NotifyStatusThroughChat(string message)
             stringPayload.value = message;
             SendMessage("NotifyStatusThroughChat", stringPayload);
         }
+        public static void ReportVideoStartedEvent(float videoPosition)
+        {
+            SendVideoStartedEvent sendVideoStartedEvent = new SendVideoStartedEvent()
+            {
+                currentPosition = videoPosition
+            };
+            
+            SendMessage("VideoStartedEvent", sendVideoStartedEvent);
+        }
+        public static void ReportVideoPausedEvent(float videoPosition)
+        {
+            SendVideoPausedEvent sendVideoStartedEvent = new SendVideoPausedEvent()
+            {
+                currentPosition = videoPosition
+            };
+            
+            SendMessage("VideoPausedEvent", sendVideoStartedEvent);
+        }
```

Comment:
Can this be replaced by the `ReportVideoProgressEvent`? Embedding all the data of the video in a single atomic message is better for the data-oriented approach of the ECS, it would add a component to the video that has a consistent full state all the times. What do you think?

The package is also missing the SceneId and the componentId of the video texture, that is necessary to redirect the event directly to the scene